.github/workflows/ci.yml

```
@@ -8,6 +8,9 @@ on:
   schedule:
     - cron: "0 12 * * *"
 
+permissions:
+  contents: read
```

You might be able to get away with dropping perms entirely:

```suggestion
permissions: {}
```